src/main/java/pages/BasePage.java

```diff
@@ -32,8 +34,8 @@ public class BasePage {
     public By newsletterInputLocator = By.xpath(footersocialXpath + "//input[@id='appendedInputButton']");
     public By newsletterButtonLocator = By.xpath(footersocialXpath + "//button[@class='btn btn-orange']");
 
-    public BasePage(WebDriver driver) {
-        this.driver = driver;
+    public BasePage(BrowserName browserName) {
```

Comment:
Maybe we can just take the browserName from the config instead of passing it here?

Reply:
@Fishgame1 This has now been done in the second latest commit